daemon/kmd/config/config.go

```diff
@@ -54,6 +55,11 @@ type SQLiteWalletDriverConfig struct {
 	ScryptParams ScryptParams `json:"scrypt"`
 }
 
+// LedgerWalletDriverConfig is configuration specific to the LedgerWalletDriver
+type LedgerWalletDriverConfig struct {
+	Disable bool `json:"disable"`
```

Comment:
How was this disabled previously?

Reply:
I didn't see a way to disable them before. We only had to do this for circle because they don't allow you to scan usbs on the osx agents

Reply:
If you needed this, I guess it means kmd crashes if HID is not supported. We should probably fix that instead of disabling KMD on circle. Or at least create an issue for fixing the issue and removing this workaroud.

Reply:
While digging into it with @bricerisingalgorand and it seems like without forking the https://github.com/karalabe/hid project and modifying it to use a new fork of the https://github.com/signal11/hidapi project it would be hard to fix the issue, the crashes are happening deep in some C code embedded in karalabe/hid. Some related tickets
https://github.com/signal11/hidapi/issues/205
https://github.com/signal11/hidapi/issues/337

In general, the ability to disable USB probing for a ledger via configuration seemed like a useful feature anyway, and we haven't seen this crash from users, so it seemed like a pretty specific corner case related to the way the CI VMs are set up.

Reply:
yikes, too bad. I was hoping it would be a matter of checking for this error:
```
func LedgerEnumerate() ([]hid.DeviceInfo, error) {
	if !hid.Supported() {
		return nil, fmt.Errorf("HID not supported")
	}
```

Reply:
sadly supported is just a `return true` https://github.com/karalabe/hid/blob/master/hid_enabled.go#L61-L66 based on whether you have the right platform-specific build tag enabled or not. the crash was coming from hid.Enumerate